### PR TITLE
Solves issue #120 and some minor changes.

### PR DESCRIPTION
**Pull request by garimasingh128**
- Created a customisable name logo for the header in portfolio.js.
- The social media icons won't show up if their links are not provided by the user.
- Fixed some minor spelling and grammatical mistakes.

### CODE REVIEW

**Comment by naveen521kk** [comment]
Can you add social media icon that we know like Instagram, Twitter?

**Comment by garimasingh128** [comment]
Do I have to add them in socialmedialinks component?

**Comment by naveen521kk** [comment]
Yeah. They needed to be added in the social media component so that the user may also enter their own links. Further adding hover effects like other things do is also great. Just do for Twitter, Instagram and we are good to go.

File: src/portfolio.js [hunk 1]
```diff
@@ -20,14 +21,14 @@ const socialMediaLinks = {
   linkedin: "https://www.linkedin.com/in/saadpasta/",
   gmail: "saadpasta70@gmail.com",
   gitlab: "https://gitlab.com/saadpasta",
-  facebook: "https://www.facebook.com/saad.pasta7"
+  facebook: "https://www.facebook.com/saad.pasta7",
```
Comment (naveen521kk) [inline src/portfolio.js:24]: Add a comment saying that Instagram and twitter is also supported.

**Comment by naveen521kk** [comment]
@allcontributors please add @garimasingh128  for code

**Comment by kartik918** [comment]
@naveen521kk please squash and merge every PR as not doing so populates the master graph with failing commits. Thanks!

**Pull request merged**